library/swift_disk.py

```diff
@@ -79,4 +78,4 @@ def main():
 
 # this is magic, see lib/ansible/module.params['common.py
 #<<INCLUDE_ANSIBLE_MODULE_COMMON>>
-main()
+main()
```

Comment:
Can you take out the changes you made here? There should be no reason for this file to change in this PR.